test/controllers/api/v1/widget_datapoints_controller_test.rb

```diff
@@ -4,7 +4,7 @@ class WidgetDatapointsControllerTest < ActionDispatch::IntegrationTest
   include Rack::Test::Methods
 
   setup do
-    create(:widget_datapoint)
+    create(:widget_datapoint, buffalo_number: 1)
```

Comment:
@lbraun looks like maybe this test was broken? The api endpoint filters datapoints based field_name (`buffalo_number is not null`). It seems like we need to set the attribute here and then expect it in the response.